packages/yarnpkg-pnpify/sources/generateSdk.ts

```diff
@@ -3,34 +3,30 @@ import CJSON                                       from 'comment-json';
 
 import {dynamicRequire}                            from './dynamicRequire';
 
-const TEMPLATE = (relPnpApiPath: string, module: string, {usePnpify}: {usePnpify: boolean}) => [
+const TEMPLATE = (relPnpApiPath: PortablePath, module: string, {usePnpify}: {usePnpify: boolean}) => [
+  `const {createRequire, createRequireFromPath} = require(\`module\`);\n`,
   `const {dirname, resolve} = require(\`path\`);\n`,
   `\n`,
-  `const relPnpApiPath = ${JSON.stringify(npath.toPortablePath(relPnpApiPath))};\n`,
+  `const relPnpApiPath = ${JSON.stringify(npath.fromPortablePath(relPnpApiPath))};\n`,
   `\n`,
   `const absPnpApiPath = resolve(__dirname, relPnpApiPath);\n`,
-  `const absRootPath = dirname(absPnpApiPath);\n`,
+  `const absRequire = (createRequire || createRequireFromPath)(absPnpApiPath);\n`,
   `\n`,
   `// Setup the environment to be able to require ${module}\n`,
   `require(absPnpApiPath).setup();\n`,
   `\n`,
-  ...(usePnpify ? [
-    `const pnpifyResolution = require.resolve(\`@yarnpkg/pnpify\`, {paths: [absRootPath]});\n`,
-  ] : []),
-  `const moduleResolution = require.resolve(\`${module}\`, {paths: [absRootPath]});\n`,
-  `\n`,
   `// Prepare the environment (to be ready in case of child_process.spawn etc)\n`,
   `process.env.NODE_OPTIONS = process.env.NODE_OPTIONS || \`\`;\n`,
   `process.env.NODE_OPTIONS += \` -r \${absPnpApiPath}\`;\n`,
   ...(usePnpify ? [
     `process.env.NODE_OPTIONS += \` -r \${pnpifyResolution}\`;\n`,
```

Comment:
`pnpifyResolution` doesn't exist anymore

Reply:
Indeed, will fix that. Fortunately none of the tools in the SDK require PnPify anymore - I'll still keep it just in case.